Updated Universal Connector Doc (#6210)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/universal-connector-source.md

  <div>If you leave the time parsing configuration blank, it will cause the source to use current time for the collected logs. Be sure to configure the HTTP response log ingestion configuration section to ensure time parsing is correctly handled. The source will enter an error health status if time parsing is configured and is unsuccessful.</div>
</details>

:::note

> This source supports up to **512 MB** of memory by default in single API response. For further requirements please contact the **Sumo Logic Support**.

Reviewer: ```suggestion
By default, this source supports up to *512 MB* of memory in a single API response. For higher limits or additional requirements, contact Sumo Logic Support.
```
